packages/cli-hermes/src/profileHermes/index.ts

```diff
@@ -0,0 +1,79 @@
+import {logger} from '@react-native-community/cli-tools';
+import {Config} from '@react-native-community/cli-types';
+import {downloadProfile} from './downloadProfile';
+
+type Options = {
+  verbose: boolean;
+  fileName?: string;
+  raw?: boolean;
+  sourcemapPath?: string;
+  generateSourcemap?: boolean;
+};
+
+async function profileHermes(
+  [dstPath]: Array<string>,
+  ctx: Config,
+  options: Options,
+) {
+  try {
+    logger.info(
+      'Downloading a Hermes Sampling Profiler from your Android device...\n',
```

Comment:
Similar to comment in other places, we don't use `\n` in `logger` anywhere else. If that was for styling purposes, I would welcome a separate PR to improve logging experience for all output 👯

Reply:
Yeah the reason why I put a new line after each sentence is that it loos very clustered, so I just wanted to give some space after each log 😆